package.json

```diff
@@ -11,43 +11,43 @@
         "lint": "npx eslint . --ext .js,.jsx,.ts,.tsx"
     },
     "devDependencies": {
-        "@aws-cdk/assert": "1.119.0",
-        "@types/jest": "^26.0.15",
-        "@types/js-yaml": "^3.12.5",
-        "@types/node": "^10.17.27",
-        "@typescript-eslint/eslint-plugin": "^4.23.0",
-        "@typescript-eslint/parser": "^4.23.0",
-        "aws-cdk": "^1.119.0",
+        "@aws-cdk/assert": "1.124.0",
+        "@types/jest": "^27.0.2",
+        "@types/js-yaml": "^4.0.3",
+        "@types/node": "^16.10.1",
+        "@typescript-eslint/eslint-plugin": "^4.32.0",
+        "@typescript-eslint/parser": "^4.32.0",
+        "aws-cdk": "^1.124.0",
         "copyfiles": "^2.4.1",
-        "eslint": "^7.26.0",
-        "jest": "^26.6.3",
-        "ts-jest": "^26.4.3",
-        "ts-node": "^8.1.0",
-        "typescript": "~3.9.7"
+        "eslint": "^7.32.0",
+        "jest": "^27.2.2",
+        "ts-jest": "^27.0.5",
+        "ts-node": "^10.2.1",
+        "typescript": "~4.4.3"
     },
     "dependencies": {
-        "@aws-cdk/aws-autoscaling": "1.119.0",
-        "@aws-cdk/aws-certificatemanager": "1.119.0", 
-        "@aws-cdk/aws-codebuild": "1.119.0",
-        "@aws-cdk/aws-codecommit": "1.119.0",
-        "@aws-cdk/aws-codepipeline": "1.119.0",
-        "@aws-cdk/aws-codepipeline-actions": "1.119.0",
-        "@aws-cdk/aws-dynamodb": "1.119.0",
-        "@aws-cdk/aws-ec2": "1.119.0",
-        "@aws-cdk/aws-ecr": "1.119.0",
-        "@aws-cdk/aws-eks": "1.119.0",
-        "@aws-cdk/aws-elasticsearch": "1.119.0",
-        "@aws-cdk/aws-iam": "1.119.0",
-        "@aws-cdk/aws-logs": "1.119.0",
-        "@aws-cdk/aws-secretsmanager": "1.119.0",
-        "@aws-cdk/core": "1.119.0",
-        "@aws-cdk/pipelines": "1.119.0",
+        "@aws-cdk/aws-autoscaling": "1.124.0",
+        "@aws-cdk/aws-certificatemanager": "1.124.0", 
+        "@aws-cdk/aws-codebuild": "1.124.0",
+        "@aws-cdk/aws-codecommit": "1.124.0",
+        "@aws-cdk/aws-codepipeline": "1.124.0",
+        "@aws-cdk/aws-codepipeline-actions": "1.124.0",
+        "@aws-cdk/aws-ec2": "1.124.0",
+        "@aws-cdk/aws-ecr": "1.124.0",
+        "@aws-cdk/aws-eks": "1.124.0",
+        "@aws-cdk/aws-iam": "1.124.0",
+        "@aws-cdk/aws-kms": "1.124.0",
+        "@aws-cdk/aws-logs": "1.124.0",
+        "@aws-cdk/aws-s3": "1.124.0", 
+        "@aws-cdk/aws-secretsmanager": "1.124.0",
+        "@aws-cdk/aws-route53": "1.124.0", 
+        "@aws-cdk/core": "1.124.0",
+        "@aws-cdk/pipelines": "1.124.0",
         "@types/bcrypt": "^5.0.0",
-        "aws-sdk": "^2.952.0",
+        "aws-sdk": "2.996.0",        
```

Comment:
Is there whitespace here after `,`?

Reply:
need to set vscode to removing trailing whitespace. :)
addressed